packages/cli/src/ceramic-daemon.ts

```diff
@@ -299,6 +302,10 @@ export class CeramicDaemon {
     documentsRouter.getAsync('/:docid', this.stateOld.bind(this)) // Deprecated
     recordsRouter.getAsync('/:streamid', this.commits.bind(this)) // Deprecated
     collectionRouter.getAsync('/', this.getCollection.bind(this))
+    adminModelRouter.getAsync('/', this.getIndexedModels.bind(this))
+    adminModelRouter.postAsync('/', this.addModelsToIndex.bind(this))
```

Comment:
`startIndexingModels`, `stopIndexingModels, and `replaceIndexedModels` to match the names of the AdminApi methods?